docs/spec/spec.md

```diff
@@ -0,0 +1,379 @@
+# Specification: Ballerina WebSubHub Library
+
+_Owners_: @shafreenAnfar @chamil321 @ayeshLK    
+_Reviewers_: @shafreenAnfar    
+_Created_: 2021/11/23  
+_Updated_: 2021/11/23  
+_Issue_: [#2162](https://github.com/ballerina-platform/ballerina-standard-library/issues/2162)
```

Comment:
I think this issue is more relevant than the above. Maybe we can update the issue with a comment saying final design has evolved from the original to the current one.

https://github.com/ballerina-platform/ballerina-standard-library/issues/786

Reply:
Fixed with [eb6cf1df06216511ce28c833508ef1c300649bcf](https://github.com/ballerina-platform/module-ballerina-websubhub/pull/378/commits/eb6cf1df06216511ce28c833508ef1c300649bcf)